backend/fittoring/src/main/java/fittoring/application/notification/presentation/NotificationController.java

```diff
@@ -0,0 +1,26 @@
+package fittoring.application.notification.presentation;
+
+import fittoring.application.notification.presentation.dto.request.FcmTokenUpsertRequest;
+import fittoring.application.notification.service.MemberFcmTokenService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RequestMapping("/notification")
+@RestController
+public class NotificationController {
+
+    private final MemberFcmTokenService memberFcmTokenService;
+
+    @PostMapping("/token")
```

Comment:
p2: 복수형으로 표현하는 것이 좋을 것 같아요! 
```suggestion
    @PostMapping("/tokens")
```